app/controllers/media_controller.rb

```diff
@@ -19,14 +19,24 @@ def stream
     authorize! :read, @media
     respond_to do |format|
       format.m3u8 do
-        redirect_to @media.to_playlist_url
+        redirect_to "#{@media.to_playlist_url}?token_string=#{media_token}"
       end
       format.mpd do
-        redirect_to @media.to_manifest_url
+        redirect_to "#{@media.to_manifest_url}?token_string=#{media_token}"
       end
     end
   end
 
+  # rubocop:disable Metrics/LineLength
+  def verify_token
+    if StacksMediaToken.verify_encrypted_token_string params[:token_string], params[:id], params[:file_name], params[:user_ip_addr]
```

Comment:
if params[:blah]  is used in multiple places in the file, it may make sense to pull out a private method

```
def file_name
  @fname ||= (allowed_)params[:file_name]
end
```
which could also alleviate the line that is too long.

Another way to alleviate that long line would be to make the conditional statement a private method?